pkg/scuzz/control.go

```diff
@@ -4,11 +4,43 @@
 
 package scuzz
 
-// Disk is the interface to a disk, with operations
-// to create packets and operate on them.
-type Disk interface {
-	// Unlock unlocks the drive, given a password
-	Unlock(string, uint, bool) error
-	// Identify returns drive identity information
-	Identify(timeout uint) error
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type (
```

Comment:
super duper nit: grouped type statements always looked weird to me

Reply:
ack.